### PR TITLE
Update erroneous and terrible README

### PR DESCRIPTION
**Pull request by himmelmaus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # cook-book
-CompSoc Cook Book! Hey, good looking, whatcha got cooking?
```
Comment (qaisjp) [inline README.md:2]: https://youtu.be/bjCoKslQOEs?t=18

**Comment by neanias** [comment]
Irrelevant

**Comment by qaisjp** [comment]
just like you

**Comment by qaisjp** [comment]
and me

**Pull request closed without merging**